acceptfunc.go

```diff
@@ -52,3 +57,43 @@ func defaultMsgAcceptFunc(dh Header) MsgAcceptAction {
 	}
 	return MsgAccept
 }
+
+type PacketAction int
```

Comment:
I think we should just reuse `MsgAcceptAction` here. The options (Accept, Reject, Ignore) seem like the exact same options we'd want here.